services/rook-ceph-cluster/1.11.4/defaults/cm.yaml

```diff
@@ -13,6 +13,7 @@ data:
       # The name is hardcoded, so if deploying more than one `rook-ceph-custer` then this flag needs to be set to false.
       # This is enabled by default to workaround D2IQ-96634
       enabled: true
+      priorityClassName: system-cluster-critical
```

Comment:
this could probably have a lower priority as its just meant for debugging ceph components, but either way works

Reply:
sounds good to me 👍